lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
```

Comment:
`const` is for things that won't change and `let` is for things that are intended to be scoped to a block, not a function